dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -54,6 +56,8 @@ internal class SdkFeature(
 ) : FeatureScope {
 
     internal val initialized = AtomicBoolean(false)
+    internal val contextUpdateListeners: ConcurrentHashMap<FeatureContextUpdateListener, String> =
+        ConcurrentHashMap()
```

Comment:
Seems like it is used like a `Set`, and value is actually never used. In this case it can be just `Collections.newSetFromMap(ConcurrentHashMap())` and placeholder for he value is not needed.

Reply:
yes, wanted to use the same thing but not available for API 21 :(

Reply:
It should be available https://developer.android.com/reference/java/util/Collections#newSetFromMap(java.util.Map%3CE,java.lang.Boolean%3E)

Reply:
ah yes...that's not what I tested, I tested the `Collections.newKeySet` which is a new method, but indeed that one would work also. Tks !!